src/proposals/20230913/SparkEthereum_20231129.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.10;
+
+import { SparkPayloadEthereum } from '../../SparkPayloadEthereum.sol';
+
+/**
+ * @title  November 29, 2023 Spark Ethereum Proposal - Set DAI supply spread to 0.5%
+ * @author Phoenix Labs
+ * @dev    This proposal updates DAI Interest Rate Strategy supplySpread parameter to match borrowSpread parameter.
+ * Forum:           TODO
```

Comment:
Forum post is up: https://forum.makerdao.com/t/accounting-discrepancy-in-the-dai-market/22845

Reply:
Updated

Reply:
Updated